sophosendpointprotection.json

```diff
@@ -0,0 +1,2207 @@
+          "example_values": [
+            0
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.os.isServer",
+          "data_type": "boolean",
+          "example_values": [
+            true,
+            false
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.os.majorVersion",
+          "data_type": "numeric",
+          "example_values": [
+            0
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.os.minorVersion",
+          "data_type": "numeric",
+          "example_values": [
+            0
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.os.name",
+          "data_type": "string",
+          "column_name": "OS Name",
+          "column_order": 2
+        },
+        {
+          "data_path": "action_result.data.*.os.platform",
+          "data_type": "string",
+          "column_name": "Platform",
+          "example_values": [
+            "windows"
+          ],
+          "column_order": 3
+        },
+        {
+          "data_path": "action_result.data.*.tamperProtectionEnabled",
+          "data_type": "boolean",
+          "example_values": [
+            true,
+            false
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.tenant.id",
+          "data_type": "string",
+          "example_values": [
+            "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+          ]
+        },
+        {
+          "data_path": "action_result.data.*.type",
+          "data_type": "string",
+          "example_values": [
+            "computer"
+          ]
+        },
+        {
+          "data_path": "action_result.summary.total_number_of_items",
+          "data_type": "numeric",
+          "example_values": [
+            1
+          ]
+        },
+        {
+          "data_path": "action_result.message",
+          "data_type": "string"
+        },
+        {
+          "data_path": "summary.total_objects",
+          "data_type": "numeric",
+          "example_values": [
+            1
+          ]
+        },
+        {
+          "data_path": "summary.total_objects_successful",
+          "data_type": "numeric",
+          "example_values": [
+            1
+          ]
+        }
+      ],
+      "render": {
+        "type": "table"
+      },
+      "versions": "EQ(*)"
+    },
+    {
+      "action": "check updates",
+      "identifier": "check_updates",
+      "description": "Send a request to the endpoint to check for Sophos management agent software updates",
+      "type": "contain",
+      "read_only": false,
+      "parameters": {
+        "endpointid": {
+          "description": "The endpoint ID",
+          "data_type": "string",
+          "required": true,
+          "order": 0
+        }
+      },
+      "output": [
+        {
+          "data_path": "action_result.status",
+          "data_type": "string",
+          "column_name": "Status",
```

Comment:
Please remove this line as we are displaying `action_result.data.*.status` datapath in the table view.